pylops/transforms/discreetcosine.py

```diff
@@ -0,0 +1,71 @@
+__all__ = ["DiscreetCosine"]
+
+from typing import Optional, Union
+
+import numpy as np
+from scipy import fft
+
+from pylops import LinearOperator
+from pylops.utils._internal import _value_or_sized_to_tuple
+from pylops.utils.typing import DTypeLike, InputDimsLike, NDArray
+from pylops.utils.decorators import reshaped
+
+class DiscreetCosine(LinearOperator):
+    r"""Discreet Cosine Transform
+
+    Performs discreet cosine transform on the given multi-dimensional
+    array along the given axis.
+
+    Parameters
+    ----------
+    dims : :obj:`list` or :obj:`int`
+        Number of samples for each dimension
+    axis : :obj:`int`, optional
```

Comment:
Mmh, the scipy code you call inside the operator allows passing multiple values to their axes. Why limit ourselves to only 1 here?

Also why not exposing also type so that with one operator we allow all types of DCT

Reply:
Similarly I would also expose workers :)

Reply:
added workers, types in dct [1542bfc](https://github.com/PyLops/pylops/pull/480/commits/1542bfcb2814a2d7a90ab9eabf72ea99b6c8f597)